data/installed-tests/fwupdtool.sh

```diff
@@ -9,10 +9,15 @@ INPUT="@installedtestsdir@/fakedevice124.bin \
 
 # ---
 echo "Building ${CAB}..."
+rm -f ${CAB}
 fwupdtool build-cabinet ${CAB} ${INPUT}
```

Comment:
this feels like we should teach build-cabinet about `--force` to allow overwriting.